weeks/week02/week02.md

```diff
@@ -56,7 +56,7 @@ These tasks should be delivered/solved in maximum 1 week (next Friday):
 - [1041: Point Location](https://judge.beecrowd.com/en/problems/view/1041)
 - [1071: Sum of Consecutive Integers](https://judge.beecrowd.com/en/problems/view/1071)
 - [1094: Experiment](https://judge.beecrowd.com/en/problems/view/1094)
-- [1085: Chess](https://judge.beecrowd.com/en/problems/view/1085)
+- [1085: Babel](https://judge.beecrowd.com/en/problems/view/1085)
```

Comment:
Let's delete it from the list. It is about Graph. Need deeper knowledge and that's not the focus.